features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -716,3 +724,40 @@ Feature: Gift Exchange Challenge
       And I uncheck "exchange_collection (recip)"
       And I press "Post"
     Then I should see "For recip."
+
+  Scenario: User sidebar assignments count only includes unfulfilled assignments.
+    Given the user "recip" exists and is activated
+      And I am logged in as "gifter"
+      And "gifter" has an assignment for the user "recip" in the collection "exchange_collection"
+    When I go to the assignments page for "gifter"
+    Then I should see "Assignments (1)" within "#dashboard"
+    When I fulfill my assignment
+      And I go to the assignments page for "gifter"
+      Then I should see "Assignments (0)" within "#dashboard"
+
+  Scenario: Many Assignments.
+    Given the following activated users exist
+      | login  |
+      | recip1 |
+      | recip2 |
+      | recip3 |
+      | recip4 |
+      | recip5 |
+      And there are 3 assignments per page
```

Comment:
It would be fine to cut this down to one per page to make this a little shorter.